src/components/partner/dto/update-partner.dto.ts

```diff
@@ -49,6 +50,14 @@ export abstract class UpdatePartner {
   @NameField({ nullable: true })
   readonly address?: string;
 
+  @Field(() => String, { nullable: true })
+  @Transform(({ value: str }) => (str ? externalUrlWithProtocol(str) : null))
```

Comment:
Kinda sucks to have this in so many spots. Maybe we move into `UrlField`? Maybe it could be conditional based on the `require_protocol` field?